pennylane/qaoa/cycle.py

```diff
@@ -155,60 +155,66 @@ def cycle_mixer(graph: nx.DiGraph | rx.PyDiGraph) -> Operator:
     >>> import networkx as nx
     >>> g = nx.complete_graph(3).to_directed()
     >>> h_m = cycle_mixer(g)
-    >>> print(h_m)
-      (-0.25) [X0 Y1 Y5]
-    + (-0.25) [X1 Y0 Y3]
-    + (-0.25) [X2 Y3 Y4]
-    + (-0.25) [X3 Y2 Y1]
-    + (-0.25) [X4 Y5 Y2]
-    + (-0.25) [X5 Y4 Y0]
-    + (0.25) [X0 X1 X5]
-    + (0.25) [Y0 Y1 X5]
-    + (0.25) [Y0 X1 Y5]
-    + (0.25) [X1 X0 X3]
-    + (0.25) [Y1 Y0 X3]
-    + (0.25) [Y1 X0 Y3]
-    + (0.25) [X2 X3 X4]
-    + (0.25) [Y2 Y3 X4]
-    + (0.25) [Y2 X3 Y4]
-    + (0.25) [X3 X2 X1]
-    + (0.25) [Y3 Y2 X1]
-    + (0.25) [Y3 X2 Y1]
-    + (0.25) [X4 X5 X2]
-    + (0.25) [Y4 Y5 X2]
-    + (0.25) [Y4 X5 Y2]
-    + (0.25) [X5 X4 X0]
-    + (0.25) [Y5 Y4 X0]
-    + (0.25) [Y5 X4 Y0]
+    >>> from pprint import pprint
+    >>> pprint(h_m)
```

Comment:
So actually the repr looks like? But this probably means we need to improve the `str` of `Sum` objects. Currently it looks like we have it where `repr` is the readable one.

```suggestion
    >>> h_m
```